content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md

```diff
@@ -0,0 +1,481 @@
+
+![Cloud-connected Intruder Detector using a Nicla Voice and a Portenta H7](assets/system_setup.png)
+
+In such a connected world, where we have a lot of information in the palm of our hand, it would also be very useful to have the status of those things that we have insured in a more analogous way, such as the front door of our home. Leveraging the integrated microphone of the Nicla Voice provides the possibility to build this kind of protection system.
+
+## Goals
+
+The goal of this application note is to showcase an intruder detection and monitoring system for the front door of the house using a combination of Nicla Voice and Portenta H7 as a host, and the Arduino IoT Cloud. The project's objectives are the following:
+
+- Analyze the surrounding sounds with the onboard microphone of the Nicla Voice and run artificial intelligence algorithms at the edge.
+- Correctly identify if a door has been opened in an ordinary manner or suffered a forced attempt.
+- Report opening and intruder detection events through Bluetooth® Low Energy to a Host.
+- The host must forward the data from the Nicla Voice to the Arduino IoT cloud using an Internet connection (Wi-Fi® in this case).
+- Create an Arduino Cloud dashboard that syncs in real time to inform and alert the user.
+- Gather real-world data to train a Machine Learning model.
+
+## Hardware and Software Requirements
+
+![Nicla Voice, Portenta H7, and 3D enclosures](assets/hardware.png)
+
+### Hardware Requirements
+- Nicla Voice
+- 3.7 V LiPo battery
+- Portenta H7
+- USB-C® cable
+- Micro USB cable
+- 5 VDC wall power adapter
+- DC brushless fan Model: LD3007MS
+- [Nicla Voice and Portenta H7 3D Printed enclosures](assets/Intruder_Detector_3D_Files.zip)
+
+### Software Requirements
+- [Arduino IDE 1.8.10+](https://www.arduino.cc/en/software), [Arduino IDE 2.0+](https://www.arduino.cc/en/software), or [Arduino Web Editor](https://create.arduino.cc/editor)
+- If you are going to use an offline Arduino IDE, you must install the following libraries: `ArduinoBLE`, `ArduinoIoTCloud`, and `Arduino_ConnectionsHandler`. You can install them using the Arduino IDE Library Manager.
+- The [intruder detector example code](assets/Nicla_Portenta_Codes.zip)
+- [Edge Impulse® CLI](https://docs.edgeimpulse.com/docs/edge-impulse-cli/cli-overview), which you can install following these [instructions](https://docs.edgeimpulse.com/docs/edge-impulse-cli/cli-installation)
+- Nicla Voice audio [firmware](https://cdn.edgeimpulse.com/firmware/arduino-nicla-voice-firmware.zip) to upload recordings to Edge Impulse®.
+- [Syntiant® uploader and precompiled ML model](assets/Syntiant_Uploader.zip)
+- [Arduino Agent](https://create.arduino.cc/getting-started/plugin/welcome) to add the Portenta H7 to the IoT Cloud
```

Comment:
The right name is "Arduino Create Agent"

Reply:
indeed, thank you for the catch!